common/exception/src/exception.rs

```diff
@@ -149,6 +150,28 @@ build_exceptions! {
     NotFoundSession(44),
     CannotListenerPort(45),
 
+    // user api error codes
+    UserNotFound(300),
+    UserAlreadyExists(301),
+
+
+    // metastore error codes
+
+    DatabaseNotFound(400),
```

Comment:
There is another error `UnknownDatabase`.
Are these two means the same?

Reply:
I did not notice that, let me remove 400